.husky/pre-push

```diff
@@ -2,4 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 CI=true npm run ci
-npm run test:e2e
+CI=true npm run test:e2e
```

Comment:
Oh ok, that explains it then. Thanks.

Reply:
If you run the following command locally, does it successfully quit?

```
      npx concurrently -k -s first -n "SB,TEST" -c "magenta,blue" \
            "npx http-server storybook-static --port 6006 --silent" \
            "npx wait-on tcp:6006 && npm run test:storybook"
```

This is what storybook does with small changes to use our scripts - and i couldn't get this working on prepush or mac ci as it didn't exit gracefully. That being said - I imagine you ran it locally fine because you had two seperate processes. Did you ever run it with the same type of "single command" and successfully exit? 

I'm not 100% keen on adding a ton of things to "pre-push" myself, I don't even love the e2e's running but if we feel its a super hard requirement i'm happy to keep looking.

I got the command from here: https://storybook.js.org/tutorials/ui-testing-handbook/react/en/automate/

Reply:
The command didn't work at all for me:

```
david.adams@C02G31Q7MD6V forgerock-web-login-framework % npx concurrently -k -s first -n "SB,TEST" -c "magenta,blue" \
            "npx http-server storybook-static --port 6006 --silent" \
            "npx wait-on tcp:6006 && npm run test:storybook"
[TEST] 
[TEST] > @forgerock/login-framework@0.0.1 test:storybook
[TEST] > test-storybook
[TEST] 
[SB] (node:60671) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
[SB] (Use `node --trace-deprecation ...` to show where the warning was created)
[TEST] [test-storybook] It seems that your Storybook instance is not running at: http://localhost:6006. Are you sure it's running?
[TEST] 
[TEST] If you're not running Storybook on the default 6006 port or want to run the tests against any custom URL, you can pass the --url flag like so:
[TEST] 
[TEST] yarn test-storybook --url http://localhost:9009
[TEST] 
[TEST] More info at https://github.com/storybookjs/test-runner#getting-started
[TEST] npx wait-on tcp:6006 && npm run test:storybook exited with code 1
--> Sending SIGTERM to other processes..
[SB] npx http-server storybook-static --port 6006 --silent exited with code 0
```

I thought that this command would start storybook? Then after this error I thought hmm maybe it needs to be running - but then got a EADDRINUSE error.